internal/pkg/constants/constants.go

```diff
@@ -54,4 +56,8 @@ const (
 	WebhookConfigurationPolicyScopeLabelKey          = "kubewardenPolicyScope"
 	WebhookConfigurationPolicyNameAnnotationKey      = "kubewardenPolicyName"
 	WebhookConfigurationPolicyNamespaceAnnotationKey = "kubewardenPolicyNamespace"
+
+	ControllerReturnCodeSuccess       = 0
+	ControllerReturnCodeError         = 1
+	ControllerReturnCodeCAInitialized = 2
```

Comment:
I think we can remove this one, see my previous comment about exiting once the CA is updated/initialized

Reply:
On top of that, if we remove the exit status from above I think we can remove all the other constants and just revert to the initial code.